src/lib/panda/colors.ts

```diff
@@ -4,16 +4,16 @@ import { defineTokens } from "@pandacss/dev";
 const colors = defineTokens.colors({
   brand: {
     primary: {
-      50: { value: "#b3e2f5" },
-      100: { value: "#86d0ef" },
-      200: { value: "#6fc8ec" },
-      300: { value: "#59bfe9" },
-      400: { value: "#42b7e6" },
-      500: { value: "#1da2d8", description: "Primary brand color" },
-      600: { value: "#1780ab" },
-      700: { value: "#146f95" },
-      800: { value: "#115f7e" },
-      900: { value: "#0b3d51" },
+      50: { value: "#f0f9fc" },
+      100: { value: "#c2e6f4" },
+      200: { value: "#8bcfeb" },
+      300: { value: "#42b1de" },
+      400: { value: "#1c9fd4", description: "Primary brand color" },
```

Comment:
I'm down for setting this value to the primary brand color, but I think all of the brand colors should correspond to the same key (i.e. the other 3 brand colors are keyed at `500`). Could you refactor this so that `primary.500` is the primary brand color?

Reply:
Do you mind if I do the opposite and just refactor the other 3 real quick? I should have done that in the first place but trying to get that extra light `50` on all of them, while still keeping close to what our initial brand color palette

Reply:
essentially, my thought is that `50-400` are "light" and `500-900` are "dark" where our brand colors fall at the top of the "light".... But down to adjust for sure

Reply:
I'd rather match up with design systems from e.g. Chakra and Tailwind and have 500 be the base color (e.g. https://www.tailwindshades.com, https://tailwindcss.com/docs/customizing-colors). What we could do is something like Tailwind and/or Panda and add custom values beyond this scale. For example, go into 25 for lighter, or 950 for darker (or anything in between). Thoughts?

Reply:
Ok definitely down for that! Will do!